security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -501,30 +501,30 @@ func (sc *SecretCache) rotate(updateRootFlag bool) {
 		return
 	}
 
-	cacheLog.Debug("Refresh job running")
+	cacheLog.Debug("Rotation job running")
 
 	var secretMap sync.Map
 	wg := sync.WaitGroup{}
 	sc.secrets.Range(func(k interface{}, v interface{}) bool {
 		connKey := k.(ConnKey)
-		e := v.(model.SecretItem)
+		secret := v.(model.SecretItem)
 		logPrefix := cacheLogPrefix(connKey.ResourceName)
 
-		// only refresh root cert if updateRootFlag is set to true.
+		// only rotate root cert if updateRootFlag is set to true.
 		if updateRootFlag {
 			if connKey.ResourceName != RootCertReqResourceName {
 				return true
 			}
 
 			atomic.AddUint64(&sc.rootCertChangedCount, 1)
-			t := time.Now()
+			now := time.Now()
 			ns := &model.SecretItem{
 				ResourceName: connKey.ResourceName,
 				RootCert:     sc.rootCert,
 				ExpireTime:   sc.rootCertExpireTime,
```

Comment:
This is not related to the PR, but do we guarantee that root cert in sc.rootCert is already rotated at this point? Are there any tests covering this part?

Reply:
Good question. Can you help to check? If you find any problem, please file an issue.